Kernel/init.cpp

```diff
@@ -166,9 +166,6 @@ extern "C" UNMAP_AFTER_INIT [[noreturn]] void init()
     VMWareBackdoor::the(); // don't wait until first mouse packet
     HIDManagement::initialize();
 
-    GraphicsManagement::the().initialize();
-    ConsoleManagement::the().initialize();
-
```

Comment:
Is there a specific reason for moving graphics and console initialization later in the boot sequence? The earlier they are the more likely it is that an assertion will happen after they got initialized, which means we can output it to the screen.

Reply:
I was having some difficulties getting some things to work when they were called before the scheduler was initialized, I think it shouldn't be too hard to fix it now. I'll try to move this back before it gets merged.

Reply:
After offline discussion, we decided that while it would be nice to have the GraphicsManagement initialization in stage1, it's too difficult to do right now (VirtIOGPU and QXL will both need to respond to interrupts during init). It's been decided that this is out of scope for this PR, so I'm keeping it in.